xarray/tests/test_dataarray.py

```diff
@@ -2314,8 +2314,8 @@ def test_resample_cftimeindex(self):
                                 calendar='noleap')
         array = DataArray(np.arange(12), [('time', times)])
 
-        with raises_regex(NotImplementedError, 'to_datetimeindex'):
-            array.resample(time='6H').mean()
+        # with raises_regex(NotImplementedError, 'to_datetimeindex'):
+        array.resample(time='6H').mean()
```

Comment:
Good catch -- go ahead and remove the commented-out line too.

Reply:
You should just delete this entire test method -- you already wrote methods to test for resampling in the other file, so there's not much point in just verifying that it doesn't raise an error here.